spec/use_cases/fetch_gemfiles_spec.rb

```diff
@@ -52,4 +63,17 @@
       ])
     end
   end
+
+  context 'The gateway raised a GemfileNotFoundException' do
```

Comment:
Should also test that the result is empty so you can see that's the expected behaviour from reading the tests